test/slicing/test_convergence.py

```diff
@@ -45,7 +45,7 @@ def h(x: DataPoint) -> int:
 
 class SlicingConvergenceTest(unittest.TestCase):
     @classmethod
-    def setup_class(cls):
+    def setUpClass(cls):
```

Comment:
Did you grep for any other instances?

Reply:
it seemed to work running `pytest test/slicing`— is convention then to run `python test/slicing/test_convergence.py`?

Reply:
Did now. :) Sounds like `pytest test/.../mytest.py` works with `setup_class` or `setUpClass` but when travis runs it only `setUpClass` works. @brahmaneya @vincentschen

Reply:
travis just runs pytest so not sure why that would happen. might be due to the coverage driver? but doubt that's the reason. you can try `tox -e coverage` to see if that replicates.